source/contracts/reporting/Market.sol

```diff
@@ -63,17 +64,18 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
 
     function initialize(IReportingWindow _reportingWindow, uint256 _endTime, uint8 _numOutcomes, uint256 _numTicks, uint256 _feePerEthInAttoeth, ICash _cash, address _creator, address _designatedReporterAddress) public payable beforeInitialized returns (bool _success) {
         endInitialization();
-        require(address(_reportingWindow) != NULL_ADDRESS);
-        require(_numOutcomes >= 2);
-        require(_numOutcomes <= 8);
+        require(_numOutcomes >= 2 && _numOutcomes <= 8);
```

Comment:
Style: For bounded checks like this I prefer to have it look like:
```
require(2 <= _numOutcomes && _numOutcomes <= 8)
```
That way it looks kind of like `2 <= numOutcomes <= 8`.